tasks.py

```diff
@@ -333,15 +333,12 @@ def dev_env(ctx, architecture="amd64", name="kind", protocol=None, frr_volume_di
     else:
         run("kubectl delete po -nmetallb-system --all", echo=True)
 
-        manifests_dir = os.getcwd() + "/manifests"
+        generate_manifests(ctx)
+        manifests_dir = "config/frr" if bgp_type == "frr" else "config/native"
```

Comment:
It means having two more layers (one to kustomize native and one to kustomize frr) which I think is too much since we are also going to have more layers for the webhooks.